example/src/main/protos/com/example/foo/BUILD.bazel

```diff
@@ -1,8 +1,8 @@
 load('@rules_proto//proto:defs.bzl', 'proto_library')
 load('@rules_python//python:proto.bzl', 'py_proto_library')
-proto_library(name='c_proto', srcs=['c.proto'], visibility=['//visibility:public'])
-java_proto_library(name='c_proto_java', deps=[':c_proto'], visibility=['//visibility:public'])
-py_proto_library(name='c_proto_py', deps=[':c_proto'], visibility=['//visibility:public'])
-proto_library(name='b_proto', srcs=['b.proto'], visibility=['//visibility:public'])
+proto_library(name='b_proto', srcs=['b.proto'], deps=['//src/main/protos/com/example:a_proto'], visibility=['//visibility:public'])
 java_proto_library(name='b_proto_java', deps=[':b_proto'], visibility=['//visibility:public'])
```

Comment:
one nit-picking is that can we rename these to `c_java_proto` to be consistent with convention?

Reply:
Currently `c_proto` is generated from `c.proto`, and then `_java` gets appended because it's a secondary rule off of `proto_library`. I don't know if we can change that easily.